src/panels/lovelace/editor/config-elements/hui-tile-card-features-editor.ts

```diff
@@ -38,6 +39,7 @@ type SupportsFeature = (stateObj: HassEntity) => boolean;
 
 const FEATURE_TYPES: FeatureType[] = [
   "cover-open-close",
+  "cover-position",
```

Comment:
The naming convention for cover-features needs a rework if we introduce tilt-sliders.
Maybe the position feature should be called "cover-position-slider"?